### PR TITLE
Radar 1

### LINKED ISSUE #200: radar_1
Description: Plotting radar data using (r, theta) coordinates

NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/radar_1.ncl
NCL Plots: https://www.ncl.ucar.edu/Applications/Images/radar_1_1_lg.png
                  https://www.ncl.ucar.edu/Applications/Images/radar_1_2_lg.png

Category: Projections

### PR DESCRIPTION
**Pull request by anissa111**
Closes #200 
image
image

### CODE REVIEW

**Comment by michaelavs** [comment]
Looks great! I noticed you left the grey version as a commented out block for the user to decide. Do you think having this would be necessary overall? If not, I think it would be fine to exclude it. Otherwise, I think making the two graphs display would be the best route.

**Comment by anissa111** [comment]
> Looks great! I noticed you left the grey version as a commented out block for the user to decide. Do you think having this would be necessary overall? If not, I think it would be fine to exclude it. Otherwise, I think making the two graphs display would be the best route.

Thank you for the input! I've included a helper function so that both plots can be displayed in the sphinx docs.

File: Plots/MapProjections/NCL_radar_1.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
+"""
+NCL_radar_1.py
+===============
+This script illustrates the following concepts:
+   - Fitting radial data to a cartesian grid
+   - Creating a vertical colorbar
```
Comment (michaelavs) [inline Plots/MapProjections/NCL_radar_1.py:6]:
```suggestion
   - Creating a horizontal colorbar
```
Colorbar is horizontal not vertical

Comment (anissa111) [reply Plots/MapProjections/NCL_radar_1.py:6]: Whoops, good catch, thanks!

**Comment by anissa111** [comment]
> Thanks for adding a helper function! Only one thing to address in the code just for accuracy. I think your branch may have brought in another branch/file (box_1), if you can delete that here/locally that should resolve that issue.

Ahh, addressed! Thank you!

**Pull request merged**